spec/services/feeds/import_spec.rb

```diff
@@ -19,6 +19,18 @@
       expect(num_articles).to eq(21)
     end
 
+    it "subscribes the article author to comments", vcr: { cassette_name: "feeds_import" } do
+      described_class.call
+      article = Article.where(published_from_feed: true).first
+
+      comment_subscription = article.notification_subscriptions.where(user_id: article.user_id, config: "all_comments")
+      expect(comment_subscription).to be_present
+
+      comment_subscription_count = NotificationSubscription.where(notifiable_type: "Article",
+                                                                  config: "all_comments").count
+      expect(comment_subscription_count).to eq(21)
```

Comment:
@citizen428 (cc: @rhymes) I see what you mean. We hard code the number in other specs in the same file because we're pulling from 3rd party feeds.

I don't think that will work - that spec would fail because it changes by more than `1`. And then to make that spec pass, we'd be back to hardcoding the total of `21` as we are now.

Reply:
> I don't think that will work - that spec would fail because it changes by more than 1. 

You can leave off the number then and just assert a change happens. Or add a `from`, like `expect {}.to change(whatever).from(0)`.

Reply:
Okie dokie - updated.